site/src/components/site-menu/site-menu.tsx

```diff
@@ -87,6 +87,10 @@ export class SiteMenu {
         {
           title: 'Ionic Framework App',
           url: '/docs/guides/ionic-framework-app'
+        },
+        {
+          title: 'Firebase Push Notifications',
```

Comment:
thinking ahead to the future, maybe change the title to "Push Notifications - Firebase" or "Push Notifications with Firebase"?